src/DSP.jl

```diff
@@ -1,14 +1,23 @@
 ## DSP.jl ##
 
+## === FUNCTIONS == ##
+
 for (T, suff) in ((Float64, "D"), (Float32, ""))
 
+    """
+    Convolution between an input Vector{T} 'X', and a kernel/filter Vector{T} 'K'.
+
+    Returns: Void. Computation result is stored in 'result'.
```

Comment:
The mutating methods should still return the result (even if this was also provided as an argument), as there's no advantage to returning `Void` (my comment above was that you should change the documentation, not the function)